linux_os/guide/system/software/sap_host/accounts_authorized_local_users/rule.yml

```
@@ -23,6 +23,15 @@ rationale: |-
     requiring access to the system and application accounts for applications not installed
     on the system.
 
+identifiers:
+    cce@sle12: CCE-83195-8
+
+references:
+    disa@sle12: CCI-000366
+    nist@sle12: CM-6(b),CM-6.1(iv)
+    stigid@sle12: SLES-12-010630
+    srg@sle12: SRG-OS-000480-GPOS-00227
+
 severity: medium
 
 references:
```

```suggestion
severity: medium

references:
    disa@sle12: CCI-000366
    nist@sle12: CM-6(b),CM-6.1(iv)
    stigid@sle12: SLES-12-010630
    srg@sle12: SRG-OS-000480-GPOS-00227
```

Please merge the references section
```